README.md

```diff
@@ -1,10 +1,16 @@
 # Hands-on Lab Android Things
 
-This codelab will introduce you to the key concepts to creating your own IoT devices using the Android Things platform.  
+This codelab will introduce you to the key concepts to create your own IoT devices using the Android Things platform.
```

Comment:
Je pense surtout que "to creating" ça ne se disait pas en anglais. Mais en fait c'est pas le cas, je viens de vérifier. Donc cette modif est a revert.